src/hooks/useActions.ts

```diff
@@ -739,10 +742,18 @@ export default function useActions() {
 }
 
 async function sendIntent(
-  app,
-  fn,
-  params,
-  { ethers, from, gasLimit = GAS_LIMIT }
+  app: any,
```

Comment:
could we define the types outside the function definition to avoid those huge multiline function signatures?

Reply:
Yes its recommended. Plus the `fn: any,` its kinda enums of string, right?

You can define the type like 
```typescript
fn: 'executeAdjustment' | 'executeVote' | 'resolve'
```
and the IDE will pop-up it while write the method to choose.

Yet better way to write that its export that chunk to type so

```typescript
type AppFunctions = 'executeAdjustment' | 'executeVote' | 'resolve'
fn: AppFunctions

```
Amazing huh?

Reply:
Yes exactly what Felipe said. I didn't know the correct type so I added any, but we can improve in the future. I will add that as Felipe suggested.